edit/workload/index.vue

```diff
@@ -578,6 +578,20 @@ export default {
         }
       }
 
+      const containerResources = template.spec.containers[0].resources;
+      const nvidiaGpuLimit = containerResources?.limits[GPU_KEY];
+
+      // Though not required, requests are also set to mirror the ember ui
+      if (nvidiaGpuLimit > 0) {
+        containerResources.requests[GPU_KEY] = nvidiaGpuLimit;
```

Comment:
requests may not null - I think this needs:

`containerResources.requests = containerResources.requests || {}`